internal/tests/SchemaVersionResource/schema_version_resource_avro_test.go

```diff
@@ -0,0 +1,91 @@
+package SchemaVersionResource
+
+import (
+	. "axual.com/terraform-provider-axual/internal/tests"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestSchemaVersionAvroResource(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: GetProviderConfig(t).ProtoV6ProviderFactories,
+		ExternalProviders:        GetProviderConfig(t).ExternalProviders,
+
+		Steps: []resource.TestStep{
+			{
+				Config: GetProvider() + GetFile("axual_schema_version_avro_initial.tf"),
```

Comment:
Can we add comments to each step for clarity.

For example :  // Initial configuration for the schema and Schema version